### DISCUSSION TITLE
Proposal for improvement: "pinned repository" with longer description

### DISCUSSION THREAD

**Discussion by heidivanparys** [ISO-TC211/StandardsTracker#419] [Ideas]
A small proposal for improvement:

- make the Standards Tracker a "pinned repository" (compare e.g. https://github.com/ISO-TC211 to https://github.com/INSPIRE-MIF).
- update the description of the Standards Tracker, so it is more descriptive (a condensed version of the README file), because that is what people see when they go to https://github.com/ISO-TC211

**Comment by ronaldtse** [comment]
@heidivanparys I've just done the first!

**Comment by PeterParslow** [comment]
I've done the second - it has to be very cut down, because it can't even contain any new line characters!

**Comment by heidivanparys** [comment]
Thanks!
